src/components/popup/popup.ts

```diff
@@ -86,8 +86,10 @@ export class MPopup extends ModulVue {
     }
 
     private set propOpen(value: boolean) {
-        this.internalOpen = value;
-        this.$emit('update:open', value);
+        if (!this.disabled) {
```

Comment:
Je suggère de sortir rapidement de la fonction au lieu d'imbriquer les ifs. Par exemple
`if (this.disabled) {
   return 
}...`

Reply:
JE suis d'accord avec ça aussi.. mais ça serait l'alignement contraire de brio :/

Reply:
👍